src/xdocs/config_coding.xml

```diff
@@ -344,6 +344,19 @@ number.equals(i + j);
               href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#VARIABLE_DEF">VARIABLE_DEF</a>
             </td>
           </tr>
+          <tr>
+            <td>validateEnhancedForLoopVariable</td>
+            <td>Controls whether to check<a href = "http://docs.oracle.com/javase/specs/jls/se7/html/jls-14.html#jls-14.14.2"> enhanced for-loop </a>variable.</td>
```

Comment:
Please remove obsolete spaces from link. Change `> enhanced for-loop <` to `>enhanced for-loop<` and put spaces outside.